lib/rom/rails/model/params.rb

```diff
@@ -53,9 +53,15 @@ def self.model_name
           RUBY
         end
 
-        def timestamps
-          attribute :created_at, DateTime, default: proc { DateTime.now }
-          attribute :updated_at, DateTime, default: proc { DateTime.now }
+        def timestamps(*attrs)
+          if attrs.empty?
+            attribute :created_at, DateTime, default: proc { DateTime.now }
+            attribute :updated_at, DateTime, default: proc { DateTime.now }
+          else
+            attrs.each do |attr|
+              attribute attr, DateTime, default: proc { DateTime.now }
+            end
+          end
```

Comment:
Sure.  Also, what do you think of extracting out the conditional to a helper method, i.e. something like:
 
```ruby
def timestamp_attributes_from(*attrs)
  attrs.empty? ? DEFAULT_TIMESTAMPS : attrs
end
```
And then `timestamps` might look like this:

```ruby
def timestamps(*attrs)
  timestamp_attributes_from(*attrs).each do |attr|
    attribute attr, DateTime, default: proc { DateTime.now }
  end
end
```